content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,133 @@
+from xrpl.models.transactions import TicketCreate, AccountSet, SignerListSet, SignerEntry
+from xrpl.transaction import safe_sign_and_submit_transaction, autofill, multisign, sign
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+from xrpl.models.requests.submit_multisigned import SubmitMultisigned
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+
+# This code sample lists which tickets are outstanding against an account
+# and use the ticket for a multisignature transaction.
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/tickets.html#tickets
+# https://xrpl.org/use-tickets.html
+# https://xrpl.org/set-up-multi-signing.html#set-up-multi-signing
+# https://xrpl.org/send-a-multi-signed-transaction.html#send-a-multi-signed-transaction
+
+# Note that this will only work with xrpl-py version v2.0.0-beta.0 or later
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Generate a wallet and request faucet
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+print("Setting up all the signers' accounts via the testnet faucet, this may take a while...")
+signer_1_wallet = generate_faucet_wallet(client=client)
+signer_2_wallet = generate_faucet_wallet(client=client)
+signer_3_wallet = generate_faucet_wallet(client=client)
+
+# Set the list of accounts that are able to authorize transactions on behalf of our Account via a multi-sig transaction
+signers = [
+        SignerEntry(account=signer_1_wallet.classic_address, signer_weight=1),
+        SignerEntry(account=signer_2_wallet.classic_address, signer_weight=1),
+        SignerEntry(account=signer_3_wallet.classic_address, signer_weight=1)
+]
+
+# Display all the signers' account address
+print("\nSigners:")
+signer_int = 1
+for signer in signers:
+    print(f"{signer_int}. {signer}")
+    signer_int += 1
+
+# A multi-sig transaction is achievable by trusting a list of keys to authorize transactions on behalf of the account
+# Construct a SignerList transaction
+tx_set_signer_list = SignerListSet(
+    account=myAddr,
+    signer_quorum=3,
+    signer_entries=signers
+)
+
+# We'll set the signer_quorum to 3, each key will represent 1 signer weight
+# if 3 of the signers sign a particular transaction, it's an authorized transaction on the XRPL
+
+# Sign transaction locally and submit
+tx_set_signer_list_signed = safe_sign_and_submit_transaction(transaction=tx_set_signer_list, wallet=test_wallet, client=client)
+
+# Construct a TicketCreate transaction, 3 tickets will be created
+tx_create_ticket = TicketCreate(
+    account=myAddr,
+    ticket_count=3
```

Comment:
This makes the code easier to play around with & have things just work.
```suggestion
    ticket_count=len(signers)
```